server/character_data.py

```diff
@@ -197,13 +197,14 @@ def get_character_bookmarks(character_id):
 
 @cachetools.cached(cachetools.TTLCache(maxsize=1000, ttl=SECONDS_TO_CACHE))
 def get_character_mail(character_id):
-    mail_list = get_paged_op(
+    mail_list = get_op(
         'get_characters_character_id_mail',
         auth_id=character_id,
         character_id=character_id,
     )
     mail_dict = {entry['mail_id']: entry for entry in mail_list}
-    for mail_id, entry in mail_dict:
+    for key in mail_dict:
```

Comment:
Should be `for key, entry in mail_dict.items()`. My code had a bug where it should have said mail_dict.items(). Because we don't use key/mail_id we could instead do `for entry in mail_dict.values():`.

Reply:
ah that explains the exception it was throwing up

Reply:
done